sandbox/src/Bitcoin.tsx

```diff
@@ -232,6 +232,90 @@ function BtcSignPSBT({ bb02 }: Props) {
   );
 }
 
+function BtcSignMessage({ bb02 }: Props) {
+  const [coin, setCoin] = useState<bitbox.BtcCoin>('btc');
+  const [simpleType, setSimpleType] = useState<bitbox.BtcSimpleType>('p2wpkhP2sh');
+  const [keypath, setKeypath] = useState("m/49'/0'/0'/0/0");
+  const [message, setMessage] = useState('message');
+  const [running, setRunning] = useState(false);
+  const [result, setResult] = useState<bitbox.BtcSignMessageSignature | undefined>();
+  const [err, setErr] = useState<bitbox.Error>();
+
+  const simpleTypeOptions: bitbox.BtcSimpleType[] = ['p2wpkhP2sh', 'p2wpkh', 'p2tr'];
+
+  const script_config: bitbox.BtcScriptConfig = { simpleType };
+  const script_config_w_keypath: bitbox.BtcScriptConfigWithKeypath = { scriptConfig: script_config, keypath: "m/49'/0'/0'/0/0" };
+
+  const stringToUint8Array = (str: string) => {
+    const arr = new Uint8Array(str.length);
+    for (let i = 0; i < str.length; i++) {
+        arr[i] = str.charCodeAt(i);
+    }
+    return arr;
+  }
+
+  const parsedResult = result ? JSON.stringify(result, undefined, 2) : '';
+
+  const submitForm = async (e: FormEvent) => {
+    e.preventDefault();
+    setRunning(true);
+    setResult(undefined);
+    setErr(undefined);
+    try {
+      const signature = await bb02.btcSignMessage(coin, script_config_w_keypath, stringToUint8Array(message));
```

Comment:
you are using the hardcoded config - use `keypath` and `simpleType` to construct the one made from the user input

Reply:
Sorry, my mistake, I forgot it while testing